packages/legacy/core/App/components/buttons/HeaderHome.tsx

```diff
@@ -1,20 +1,29 @@
 import { useNavigation } from '@react-navigation/native'
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 import { useTranslation } from 'react-i18next'
 
 import { Screens, TabStacks } from '../../types/navigators'
 import { testIdWithKey } from '../../utils/testable'
 
 import IconButton, { ButtonLocation } from './IconButton'
+import { useTour } from '../../contexts/tour/tour-context'
+import { ImportantForAccessibility } from '../../types/accessibility'
 
 const HeaderRightHome: React.FC = () => {
   const { t } = useTranslation()
   const navigation = useNavigation()
+  const { currentStep } = useTour()
+  const [hideElements, setHideElements] = useState<ImportantForAccessibility>('auto')
+
+  useEffect(() => {
+    setHideElements(currentStep === undefined ? 'auto' : 'no-hide-descendants')
+  }, [currentStep])
```

Comment:
same thought here, this might be a better place for `useMemo` rather than `useEffect`. I wonder though, would it be simpler to set this `importantForAccessibility` property on the parent (header itself)? Then we wouldn't need to add it to all these children